src/data/faq_de.json

```diff
@@ -787,6 +787,16 @@
                         "textblock": [
                             "Die Details dazu finden Sie (in englischer Sprache) in unserem <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md' target='_blank' rel='noopener noreferrer'>Solution-Architecture-Dokument</a>."
                         ]
+                    },
+                    {
+                        "title": "Kontakt-Tagebuch",
+                        "anchor": "contact_diary",
+                        "active": true,
+                        "textblock": [
+                            "Seit Einführung des Kontakt-Tagebuchs erhalten wir kontinuierlich Änderungs- und Verbesserungsvorschläge. Alle diese Ideen prüfen wir technisch und gemeinsam mit dem Auftraggeber bezüglich:",
+                            "<ul><li> Nutzen für die Konsument*innen des Tagesbuchs (z.B. die Gesundheitsämter)</li><li> Anwenderfreundlichkeit ders UI,</li><li>Erweiterung durch Funktionen (z.B. Erfassen zusätzlicher Parameter),</li><li>technische Umsetzbarkeit (bzw. die Aufwände für die Implementierung) sowie</li><li>rechtlicher Rahmenbedingungen und Grenzen</li></ul>",
```

Comment:
Typo: ders 
in "Anwenderfreundlichkeit ders UI"

Ist das grammatikalisch richtig?
bezüglich: ... technische Umsetzbarkeit
oder "technischer"

Reply:
"technischer" would be correct